src/main/java/com/google/devtools/build/lib/rules/repository/NewLocalRepositoryRule.java

```diff
@@ -46,7 +47,7 @@ public RuleClass build(RuleClass.Builder builder, RuleDefinitionEnvironment envi
         named BUILD, but can be. (Something like BUILD.new-repo-name may work well for
         distinguishing it from the repository's actual BUILD files.)</p>
         <!-- #END_BLAZE_RULE.ATTRIBUTE --> */
-        .add(attr("build_file", STRING))
```

Comment:
You probably also want to update NewLocalRepositoryFuntion.resolve() because it has a branch for the case where the value of `build_file` is a string.

Reply:
done